src/kernels/InScatter.C

```diff
@@ -91,8 +91,12 @@ InScatter::computeQpOffDiagJacobian(unsigned int jvar)
     {
       if (i == _group)
         continue;
-      jac += -_test[_i][_qp] * _d_gtransfxs_d_temp[_qp][i + _group * _num_groups] * _phi[_j][_qp] *
-             computeConcentration((*_group_fluxes[i]), _qp);
+      if (_sss2_input)
```

Comment:
I'm planning to add the option to use OpenMC group constants. Will I need to touch this part? Depending on if openmc returns the scattering matrix as a transpose or not?

Reply:
Yes, moving forward, we'll have to decide whether to standardize the cross section format or to handle each case differently like the current approach